src/dashboards/components/dashboard_index/DashboardCards.tsx

```diff
@@ -18,6 +18,13 @@ import {LimitStatus} from 'src/cloud/actions/limits'
 
 // Utils
 import {extractDashboardLimits} from 'src/cloud/utils/limits'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {CLOUD} from 'src/shared/constants'
+
+let getPinnedItems
+if (CLOUD) {
+  getPinnedItems = require('src/shared/contexts/pinneditems').getPinnedItems
```

Comment:
Why do we have to use require here? why can't we just import this?

Reply:
because of the unique syntax, we do have to use require, its a pattern we have elsewhere